hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -57,13 +57,20 @@ static bool param_combined_z_axis;
 
 module_param_named(combined_z_axis, param_combined_z_axis, bool, 0644);
 MODULE_PARM_DESC(combined_z_axis,
-		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine");
+		 "(bool) Combine the triggers to form a single axis. 1: combine, 0: do not combine.");
```

Comment:
This change is unrelated and should be a separate commit, you may put it in the same PR, tho. The same for the change a few lines below.